base/operators.jl

```diff
@@ -295,10 +298,13 @@ true
 """
 ===
 const ≡ = ===
+const ⩶ = ===
 
 """
     !==(x, y)
-    ≢(x,y)
+    !⩵(x, y)
```

Comment:
This one seems a bit ambiguous to me, maybe omit it.

Reply:
`≢` was defined long before me, I just added the symbol in docstring.

I also removed the 2 more lines I should have removed yesterday!

Reply:
I was referring to `!⩵`, which is new in this PR, no?

Reply:
@stevengj Ah OK, but yes, it was of course removed (with 2 other lines) in the latest commit (about cleaning).